state-synchronizer/src/lib.rs

```diff
@@ -14,7 +18,33 @@ mod peer_manager;
 mod synchronizer;
 
 type PeerId = AccountAddress;
-type LedgerInfo = LedgerInfoWithSignatures;
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+/// The state distinguishes between the following fields:
+/// * highest_local_li is keeping the latest certified ledger info
+/// * highest_committed_version is keeping the latest version in the transaction accumulator in case
```

Comment:
the name is confusing, I always consider committed == have a LedgerInfo

Reply:
Hehe, what name would be more intuitive? `highest_applied_version`? `highest_available_version`?

Reply:
`highest_sync_version`?